lindsey-paul-challenge4/src/App.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import TextField from 'material-ui/TextField';
+import injectTapEventPlugin from 'react-tap-event-plugin';
+import FlatButton from 'material-ui/FlatButton';
+
+//injectTapEventPlugin();
+
+var contactName;
+var description;
+var price;
```

Comment:
should not need these class variables. they are also included in state. whadup with that?